### PR TITLE
[engine] Match Node requirement of Reaction

### PR DESCRIPTION
**Pull request by alloy**
Fixes this issue:

```
~/C/A/force [update-node-requirement] » nvm use
Now using node v8.11.3 (npm v5.6.0)
~/C/A/force [update-node-requirement] » yarn install
yarn install v1.10.1
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error @artsy/reaction@5.7.12: The engine "node" is incompatible with this module. Expected version "8.12.x". Got "8.11.3"
error Found incompatible module
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": "8.x.x",
+    "node": "8.12.x",
```
Comment (damassi) [inline package.json]: We can talk in the channel, but we should coordinating around  getting our apps to Node 10. cc @zephraph

**Comment by damassi** [comment]
👍 

merge on green

File: package.json (hunk 1 above)
Comment (zephraph) [reply package.json]: Yeah, we should start that convo

**Comment by zephraph** [comment]
Merge on green isn't working right now

**Pull request merged**